test/infrastructure/docker/api/v1beta1/dockercluster_types.go

```diff
@@ -88,10 +88,13 @@ type DockerClusterStatus struct {
 // APIEndpoint represents a reachable Kubernetes API endpoint.
 type APIEndpoint struct {
 	// Host is the hostname on which the API server is serving.
-	Host string `json:"host"`
+	// +optional
```

Comment:
I would also like to keep those two fields as is (godoc is fine of course, but I wouldn't add omitempty and +optional)